AsyncDisplayKit/ASEqualityHashHelpers.mm

```diff
@@ -0,0 +1,26 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#import "ASEqualityHashHelpers.h"
+
+#import <functional>
+#import <objc/runtime.h>
+#import <stdio.h>
+#import <string>
+
+NSUInteger CKIntegerArrayHash(const NSUInteger *subhashes, NSUInteger count)
```

Comment:
We should rename these functions, or otherwise retain the CK filename and place such files in a directory that designates them as unmodified CK ports.